### PR TITLE
fix: remove duplicate shub setting

### PR DESCRIPTION
**Pull request by vladdoster**

### CODE REVIEW

File: shub/settings.py
Comment (vladdoster) [inline shub/settings.py:1]: Duplicate of line [line 196](https://github.com/singularityhub/sregistry/pull/426/files?diff=unified&w=0#diff-a810a5dd30f19064fdc88124f97cca53be3734a47798cab36b26260cf2bf2822R195)

**Comment by vladdoster** [comment]
Not quite sure why the formatting check is failing...?

**Comment by vsoch** [comment]
@vladdoster add the `'.tributors` file to this line here https://github.com/singularityhub/sregistry/blob/c4c26050cb467932661409fae94a621c7b159d76/.pre-commit-config.yaml#L1 to skip checking it.

**Comment by vladdoster** [comment]
> @vladdoster add the `'.tributors` file to this line here
> 
> https://github.com/singularityhub/sregistry/blob/c4c26050cb467932661409fae94a621c7b159d76/.pre-commit-config.yaml#L1
> 
> to skip checking it.

I assume you meant to type `.tributors` and the ``` was accidental?

**Comment by vladdoster** [comment]
I'll rebase this branch once #427 is merged. I split it into a separate PR to make the changes more atomic.

**Review by vsoch — APPROVED** [review]

**Comment by vsoch** [comment]
Thanks @vladdoster !

**Pull request merged**